test_libs/pyspec/eth2spec/test/phase_0/block_processing/test_process_transfer.py

```diff
@@ -132,9 +129,6 @@ def test_insufficient_balance(spec, state):
     state.balances[sender_index] = spec.MAX_EFFECTIVE_BALANCE
     transfer = get_valid_transfer(spec, state, sender_index=sender_index, amount=1, fee=0, signed=True)
 
-    # un-activate so validator can transfer
-    state.validators[transfer.sender].activation_epoch = spec.FAR_FUTURE_EPOCH
```

Comment:
ditto

Reply:
was not sure about this one. The validator is allowed to transfer because of its balance already. But it goes under it, so actually no. Will put it back, but with proper `activation_eligibility_epoch`

Reply:
fixed in rebase